app/uk/gov/hmrc/customs/notification/repo/NotificationWorkItemRepo.scala

```diff
@@ -22,12 +22,12 @@ import com.google.inject.ImplementedBy
 import javax.inject.{Inject, Singleton}
```

Comment:
Should there be a new compound index on clientId + WorkItemStatus to support the block count query?

Reply:
Done